### ISSUE TITLE
False positive for `WPS345 Found meaningless number operation`

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#775]
# Bug report

## What's wrong

False positive for `1 / some`:

```
  13:12    WPS345 Found meaningless number operation
  return 1 / number
```

## How is that should be

`1 / ANY` should be allowed.

**Comment by serhii73** [comment]
Hello, @sobolevn. I can do it.
Can you tell me more about this task?
Because I do not fully understand when False-positive happen.
Thank you =)

**Comment by sobolevn** [comment]
@serhii73 here's the full explanation:

1. This code: https://github.com/wemake-services/wemake-python-styleguide/blob/master/wemake_python_styleguide/visitors/ast/operators.py#L98-L111 produces incorrect result. It treats `1 / some` as useless math operation. But it is not. You can and should be able to use it.
2. These tests do not check it: https://github.com/wemake-services/wemake-python-styleguide/blob/master/tests/test_visitors/test_ast/test_operators/test_useless_math.py

Things we need to do:
1. Allow `N / ` operations, they should be allowed for `N != 0`
2. Test these cases explicitly in a regression test

Thanks for your help! I appreciate that! 👍

**Comment by serhii73** [comment]
Thank you very much!
plz, assign this issue to me on the GitHub UI. Thank you =)

**Comment by serhii73** [comment]
Sorry @sobolevn can you help me?
```
@pytest.mark.parametrize('expression', [
........
])
```
I don't understand how to right write a list of expressions.
```
@pytest.mark.parametrize('expression', [
    '1 / other',
])
```
or
```
@pytest.mark.parametrize('expression', [
    '1 / 11',
])
```
it is not right.

**Comment by sobolevn** [comment]
```python
@pytest.mark.parametrize('expression', [
    '1 / other',
    '1 / 11',
])
```

Are both correct according to this rule. But:

```python
@pytest.mark.parametrize('expression', [
    'other / 1',
    '11 / 1',
])
```

Are not correct.
